src/NUnitFramework/framework/Internal/Results/TestResult.cs

```diff
@@ -52,11 +54,23 @@ public abstract class TestResult : ITestResult
         /// </summary>
         internal const double MIN_DURATION = 0.000001d;
 
-//        static Logger log = InternalTrace.GetLogger("TestResult");
+        /// <summary>
+        /// Empty child queue
+        /// </summary>
+        protected static readonly ConcurrentQueue<ITestResult> EmptyChildQueue = new ConcurrentQueue<ITestResult> ();
+
+        //        static Logger log = InternalTrace.GetLogger("TestResult");
 
         private StringBuilder _output = new StringBuilder();
         private double _duration;
 
+        /// <summary>
+        /// Aggregate assertion count
+        /// </summary>
+        protected int InternalAssertCount;
```

Comment:
This is named as a property and I don't see why you needed to add backing fields to this or the ResultState. I would prefer if we left it the way it is unless there is a reason.